### PR TITLE
Add Grafana v7.5.16

### PR DESCRIPTION
**Pull request by m1zzx2**
## What this PR does / why we need it

add grafana v7.5.16

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

File: addons/packages/grafana/7.5.16/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+### grafana Configuration
+
+| Value | Required/Optional | Description |
+|-------|-------------------|-------------|
+| `name` | Optional | description |
+
+## Usage Example
+
+The follow is a basic guide for getting started with grafana.
```
Comment (seemiller) [inline addons/packages/grafana/7.5.16/README.md:25]: This entire README.md needs to be updated to include, at a minimum, the content that is provided in the 7.5.11 version of the package.

File: addons/packages/grafana/metadata.yaml [hunk 2]
```diff
@@ -9,8 +9,8 @@ spec:
   shortDescription: "Visualization and analytics software"
   providerName: VMware
   maintainers:
-  - name: Guanpeng Gao
+  - name: XiaoShuang Guo
   categories:
     - "monitoring"
     - "observability"
-  supportDescription: "The detailed documentation is on https://grafana.com/.  Support is available from VMware. Refer to the VMware Product Guide for specific support terms. "
+  supportDescription: "The detailed documentation is on https://grafana.com/.  Support is available from VMware. Refer to the VMware Product Guide for specific support terms. "
```
Comment (seemiller) [inline addons/packages/grafana/metadata.yaml:16]: Can you add a newline at the end of this file? Otherwise the Yaml Lint check will fail.

**Review by seemiller — CHANGES_REQUESTED** [review]

File: addons/packages/grafana/7.5.16/README.md (hunk 1 above)
Comment (m1zzx2) [reply addons/packages/grafana/7.5.16/README.md:25]: thx, I will updated it later

**Comment by yuyangbj** [comment]
@ggpaue @shyaamsn could you please review this Grafana bump PR? Thanks.

File: addons/packages/grafana/7.5.16/bundle/.imgpkg/images.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: grafana/grafana:7.5.16
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: 7.5.16
+          url: grafana/grafana:7.5.16
+  image: index.docker.io/grafana/grafana@sha256:9b7b1d9a1deadbe6fed74416d58db47f8af31b5cee214cfb659f89e6dda3f716
```
Comment (yuyangbj) [inline addons/packages/grafana/7.5.16/bundle/.imgpkg/images.yml:10]: I think we should generate our own image? Refer to here image from v7.5.11: projects.registry.vmware.com/tce/grafana@sha256:ca56e9bc0e249ac006717f57f3fa1fdb3888419b224bfa305a67c28caa1364ba.

File: addons/packages/grafana/7.5.16/bundle/.imgpkg/images.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+- annotations:
+    kbld.carvel.dev/id: kiwigrid/k8s-sidecar:1.15.6
```
Comment (yuyangbj) [inline addons/packages/grafana/7.5.16/bundle/.imgpkg/images.yml:12]: @shyaamsn could you please help us understand why we should pin the version to 1.15.6?

**Review by seemiller — APPROVED** [review]

**Comment by seemiller** [comment]
Created a package repository that contains Grafana 7.5.16:

`projects-stg.registry.vmware.com/tce/main@sha256:3af4631c0f4499fe67b817add5c84a08ab82db3d09cbdb04771677ea3ab14b29`

Loosely followed the [Monitoring with Prometheus and Grafana on Docker](https://tanzucommunityedition.io/docs/v0.12/docker-monitoring-stack/) guide, except I deployed to AWS. Package installed and was accessible!

```
tanzu package repository update tce-repo --url projects-stg.registry.vmware.com/tce/main@sha256:3af4631c0f4499fe67b817add5c84a08ab82db3d09cbdb04771677ea3ab14b29

tanzu package install cert-manager --package-name cert-manager.community.tanzu.vmware.com --version 1.8.0

tanzu package install contour -p contour.community.tanzu.vmware.com --version 1.20.1

cat <<EOF > zprometheus-values.yaml
ingress:
  enabled: true
  virtual_host_fqdn: "prometheus.local"
  prometheus_prefix: "/"
  alertmanager_prefix: "/alertmanager/"
  prometheusServicePort: 80
  alertmanagerServicePort: 80
EOF

tanzu package install prometheus -p prometheus.community.tanzu.vmware.com -v 2.27.0-1 --values-file prometheus-values.yaml

tanzu package install grafana --package-name grafana.community.tanzu.vmware.com --version 7.5.16

kubectl get pods,pvc,pv,svc -n grafana

# open the external-ip from the `service/grafana` in a browser
echo `kubectl get service/grafana -n grafana -o jsonpath='{.status.loadBalancer.ingress[*].hostname}'`
```

**Pull request merged**